temba/tests.py

```diff
@@ -121,6 +121,9 @@ def setUpClass(cls):
         with open('%s/test_flows/color.json' % settings.MEDIA_ROOT, 'r') as f:
             cls.COLOR_FLOW_DEFINITION = json.load(f)
 
+        with open('%s/test_flows/flow_quick_replies.json' % settings.MEDIA_ROOT, 'r') as fqr:
+            cls.QUICK_REPLY_FLOW_DEFINITION = json.load(fqr)
```

Comment:
This doesn't need to be a class var of `TembaTest`. Color flow is a special case because we use it so often in legacy tests. Just use get_flow to load flow_quick_replies.json as and when you need it.